app/locale.js

```diff
@@ -215,6 +215,7 @@ var rendererIdentifiers = function () {
     'updateLater',
     'updateHello',
     'notificationPasswordWithUserName',
+    'notificationUpdatePasswordWithUserName',
```

Comment:
@darkdh we need `notificationUpdatePassword` along with that. Unless:

![clipboard01](https://cloud.githubusercontent.com/assets/3362943/26138541/e4dbc5f6-3b04-11e7-8dd3-599f36a385f0.jpg)